demos/src/componentBoilerplate.js

```diff
@@ -0,0 +1,6 @@
+/*global require*/
+require('../../main.js');
```

Comment:
call this file demo.js and I think the require needs a './' prefixed to the path